Makefile

```diff
@@ -18,9 +18,9 @@
 #
 
 # values used in workspace-status.sh
-DOCKER_REGISTRY?=cockroachdb
+DOCKER_REGISTRY?=us.gcr.io/chris-love-operator-playground
 DOCKER_IMAGE_REPOSITORY?=cockroachdb-operator
-VERSION?=$(shell cat version.txt)
+VERSION?=1.10.4
```

Comment:
Can we keep these as is?

Reply:
@alinadonisa please override on the command line and stop changing the Makefile

Reply:
@rail, @chrislovecnm  it is just a draft, not ready for review yet :)